frontend/store/auctions.ts

```diff
@@ -242,12 +242,12 @@ export const actions = {
         const network = rootGetters['network/getMakerNetwork'];
         const auction = getters.getAuctionById(id);
         if (!auction) {
-            message.error(`Auction reset error: can not find auction with id "${id}"`);
+            await message.error(`Auction reset error: can not find auction with id "${id}"`);
             return;
         }
         const walletAddress = getWallet().address;
         if (!walletAddress) {
-            message.error('Bidding error: can not find wallet');
+            await message.error('Bidding error: can not find wallet');
```

Comment:
Btw, what does awaiting a message implies? Does the promise gets resolved as soon as the message is shown or as soon as it's finished showing (so the promise will be resolved in 3 seconds)?

Reply:
Wow yes, it actually is a promise that resolves after the message is finished. Guess this is used for sequenced messages? But that shouldn't be awaited, right?

Reply:
> But that shouldn't be awaited, right?

Yeah, then I don't think we need to await them. Let's also use `.catch(() => {})` or `// eslint-disable-line no-floating-promises`